pallets/ethereum-events/src/lib.rs

```diff
@@ -1707,4 +1686,63 @@ pub mod migrations {
         log::info!("ℹ️  Migrated Ethereum event's NFT contract addresses successfully");
         return consumed_weight
     }
+
+    pub fn migrate_to_bridge_contract<T: Config>() -> frame_support::weights::Weight {
```

Comment:
I think this migration logic can be simplified by using a function to decide which value to use (lift_address, validator_manager_address or pub_root_address) and assign it to variable, say `avn_bridge_address_to_use` and a single block of code that does the update..
Maybe I am missing something so lets discuss offline.